nix/pkgs/libspdk/default.nix

```diff
@@ -59,8 +59,8 @@ let
     src = fetchFromGitHub {
       owner = "openebs";
       repo = "spdk";
-      rev = "0761ff891a41d348143bd27edd4d110aec3bfa2a";
-      sha256 = "sha256-vXLR9uDPZchnTMhp7jN8GGNP9RUYyTby47RixkOEpfY=";
+      rev = "4ec822d7eabb6b340021c5794c26ecc75258724b";
```

Comment:
ah does this require spdk change?

Reply:
Yes.
We relaxed changes in SPDK:  https://github.com/openebs/spdk/commit/4ec822d7eabb6b340021c5794c26ecc75258724b